austrakka/components/metadata/__init__.py

```diff
@@ -28,33 +28,61 @@ def metadata(ctx):
 @click.argument('file', type=click.File('rb'))
 @opt_proforma()
 @opt_blanks_delete()
-def submission_add(file: BufferedReader, proforma: str, blanks_will_delete: bool = False):
+@opt_batch_size(help='The number of rows to split the metadata upload into before uploading. '
+                     'If the file size is below this value, the file will not be split. '
+                     'An upload record will be recorded in the database per batch, and '
+                     'validation and success messages will be returned per batch. '
+                     'Note that if an Excel file is batched, the only the submission '
```

Comment:
There's an extra "the" in front of "only".

Reply:
This was a missed help text change as noted below!